src/extension.ts

```diff
@@ -35,14 +35,20 @@ export function activate(ctx: ExtensionContext): void {
 	}
 
 	const api = new LaunchDarklyAPI(config);
-	flagStore = new FlagStore(config, api);
+	let flagStore: FlagStore;
```

Comment:
Isn't there a `flagStore` already defined in the global scope?

Reply:
This was making things unhappy on the initial install so I changed it so that the flagStore is only created after we have API Key + Project & Environment so it had all of the necessary information to start up.